app/commands/payments/paypal/product.rb

```diff
@@ -0,0 +1,14 @@
+module Payments::Paypal::Product
+  class UnknownPaypalProductError < RuntimeError; end
+
+  def self.from_name(name)
+    case name
+    when Exercism.secrets.paypal_donation_product_name
+      :donation
+    when Exercism.secrets.paypal_premium_product_name
+      :premium
+    else
+      raise UnknownPaypalProductError
+    end
+  end
+end
```

Comment:
Same comment as on Interval.

Reply:
The other option, which maybe nicer, would be to have these as `self.product_from_name` on `Payments::Paypal` instead.

Reply:
I've refactored things a bit